packages/p2p-v2/src/components/Modals/OrderDetailsConfirmModal/OrderDetailsConfirmModal.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react';
+import { FileUploaderComponent } from '@/components/FileUploaderComponent';
+import { getErrorMessage, maxPotFileSize, TFile } from '@/utils';
+import { Button, InlineMessage, Modal, Text, useDevice } from '@deriv-com/ui';
+import './OrderDetailsConfirmModal.scss';
+
+type TOrderDetailsConfirmModalProps = {
+    isModalOpen: boolean;
+    onRequestClose: () => void;
+};
+
+type TDocumentFile = {
+    errorMessage: string | null;
+    files: TFile[];
+};
+
+const OrderDetailsConfirmModal = ({ isModalOpen, onRequestClose }: TOrderDetailsConfirmModalProps) => {
+    const [documentFile, setDocumentFile] = useState<TDocumentFile>({ errorMessage: null, files: [] });
+    const { isMobile } = useDevice();
+    const buttonTextSize = isMobile ? 'md' : 'sm';
+
+    const handleAcceptedFiles = (files: TFile[]) => {
+        if (files.length > 0) {
+            setDocumentFile({ errorMessage: null, files });
+        }
+    };
+
+    const removeFile = () => {
+        setDocumentFile({ errorMessage: null, files: [] });
+    };
+
+    const handleRejectedFiles = (files: TFile[]) => {
+        setDocumentFile({ errorMessage: getErrorMessage(files), files });
+    };
+
+    // TODO: uncomment this when implementing the OrderDetailsConfirmModal
+    // const displayPaymentAmount = removeTrailingZeros(
+    //     formatMoney(local_currency, amount_display * Number(roundOffDecimal(rate, setDecimalPlaces(rate, 6))), true)
+    // );
```

Comment:
yes, cause currently, we haven't implemented the order details where all the data is stored for the particular order, so eventually we will pass it down to this Modal 🙏  below is an example of where it would be used, and its similar to the function in order-details-confirm-modal in v1

![Screenshot 2024-03-12 at 9 15 26 AM](https://github.com/binary-com/deriv-app/assets/103412909/abf041c9-603d-4930-b104-0bf2cf67ff8e)